src/status_im/contexts/preview/quo/wallet/token_input.cljs

```diff
@@ -24,17 +25,29 @@
 
 (defn view
   []
-  (let [state (reagent/atom {:token               :eth
-                             :currency            :usd
-                             :conversion          0.02
-                             :networks            networks
-                             :title               title
-                             :customization-color :blue})]
+  (let [state     (reagent/atom {:token               :eth
+                                 :currency            :usd
+                                 :conversion          0.02
+                                 :networks            networks
+                                 :title               title
+                                 :customization-color :blue
+                                 :show-keyboard?      false})
+        value     (reagent/atom "")
+        set-value (fn [v]
+                    (swap! value str v))
+        delete    (fn [_]
+                    (swap! value #(subs % 0 (dec (count %)))))]
     (fn []
       [preview/preview-container
        {:state                     state
         :descriptor                descriptor
-        :component-container-style {:padding-horizontal 20
-                                    :margin-top         50
-                                    :align-items        :center}}
-       [quo/token-input @state]])))
+        :full-screen?              true
+        :component-container-style {:flex            1
+                                    :justify-content :space-between}}
+       [quo/token-input (assoc @state :value @value)]
+       [quo/numbered-keyboard
+        {:container-style {:padding-bottom (safe-area/get-top)}
+         :left-action     :dot
+         :delete-key?     true
+         :on-press        set-value
+         :on-delete       delete}]])))
```

Comment:
Added the `numbered-keyboard` to test this input in the preview screen.
It required to modify the `preview-container`, now receives a `full-screen?` boolean